### ISSUE TITLE
list convertion breaks - index out of range if list is empty

### ISSUE THREAD

**Issue by pinkynrg** [hgrecco/pint#758]
```
~/Projects/client-web-venv/lib/python3.7/site-packages/pint/quantity.py in wrapped(self, *args, **kwargs)
     71         # pandas often gets to arrays of quantities [ Q_(1,"m"), Q_(2,"m")]
     72         # and expects Quantity * array[Quantity] should return NotImplemented
---> 73         elif isinstance(other, list) and isinstance(other[0], type(self)):
     74             return NotImplemented
     75         result = f(self, *args, **kwargs)

IndexError: list index out of range
```

**Comment by hgrecco** [comment]
See my suggestion on #761

**Comment by hgrecco** [comment]
I have just merge a PR. Let me know if this works

**Comment by jules-ch** [comment]
Resolved with #762
